.github/workflows/ci.yml

```diff
@@ -4,12 +4,13 @@ on: push
 
 jobs:
   test:
-    name: Go ${{ matrix.go-version }} on Ubuntu
+    name: Go ${{ matrix.go-version }} with Kafka ${{ matrix.kafka-version }} on Ubuntu
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         go-version: [1.12.x, 1.13.x]
+        kafka-version: [2.3.1, 2.4.0]
```

Comment:
do you think it'd be a good idea also to test against `2.2.x` like previously travis did it with `2.2.1`?